### PR TITLE
feat: add it to web3 and nothing more

### PR DESCRIPTION
**Pull request by olizilla**
Adds a directory to web3.storage.

## Example usage
```yaml
uses: web3-storage/add-to-web3@v1
id: web3
with:
  web3_token: ${{ secrets.WEB3_STORAGE_TOKEN }}
  path_to_add: 'dist'

# "bafkreicysg23kiwv34eg2d7qweipxwosdo2py4ldv42nbauguluen5v6am"
- run: echo ${{ steps.web3.outputs.cid }}

# "https://dweb.link/ipfs/bafkreicysg23kiwv34eg2d7qweipxwosdo2py4ldv42nbauguluen5v6am"
- run: echo ${{ steps.web3.outputs.url }}
```

License: (Apache-2.0 AND MIT)
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by olizilla** [comment]
Yikes. This works in test, and in `act`, but fails in prod becuase it can't run in the `node12` runner. we use fs.promises.rm in the fs blockstore in ipfs-car which doesn't exist in node12.

```
  Rejected promise returned by test. Reason:

  TypeError {
    message: 'fs_1.default.promises.rm is not a function',
  }

  › FsBlockStore.close (node_modules/ipfs-car/dist/cjs/blockstore/fs.js:57:41)
  › Function.put (node_modules/web3.storage/src/lib.js:157:24)
  › addToWeb3 (web3.js:7:15)
  › test/test.js:20:24
```

and there is little movement on the issue to get newer node versions available https://github.com/actions/runner/issues/772

**Comment by olizilla** [comment]
IT LIVES!
```
# Just enough docker until github gets a new node16 runner
# see: https://github.com/actions/runner/issues/772
FROM node:16-alpine
COPY dist dist
ENTRYPOINT [ "node", "/dist/index.js" ]
```

**Comment by olizilla** [comment]
Now we need a fix for the extra wrapping dir... one of
- have smart default in ipfs-car https://github.com/web3-storage/ipfs-car/issues/64
- allow custom unixfs-importer params https://github.com/web3-storage/ipfs-car/pull/61
- make globs work https://github.com/web3-storage/files-from-path/issues/5

File: web3.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { getFilesFromPath } = require('files-from-path')
+const { Web3Storage } = require('web3.storage')
+
+async function addToWeb3 ({ endpoint, token, pathToAdd, name }) {
+  const web3 = new Web3Storage({ endpoint, token })
+  const files = await getFilesFromPath(`${pathToAdd}`)
```
Comment (alanshaw) [inline web3.js:6]: Not already string?

```suggestion
  const files = await getFilesFromPath(pathToAdd)
```

**Comment by olizilla** [comment]
It blends!
https://github.com/web3-storage/web3.storage/pull/108/checks?check_run_id=3147399786

**Pull request merged**